harness/determined/common/experimental/experiment.py

```diff
@@ -16,9 +21,75 @@ class is obtained via :class:`~determined.experimental.Determined`, the
             master URL is automatically passed into this constructor.
```

Comment:
ditto on Arguments.  Also please check the checkpoint directory for examples like this.